jira.xml

```diff
@@ -39,7 +38,226 @@
   </task-group>
   <task-group for=">v0.5.0">
     <issue>
-      Sep 20th: SPIKE Onboarding.
+      test what happens if there are no keepers available and initialized
```

Comment:
Note to self: some of these should be done BEFORE merging this PR, and some "after", specify which one is which.

Also this PR likely requires a minor version bump again.